typing_extensions/README.rst

```diff
@@ -31,49 +35,62 @@ Included items
 
 This module currently contains the following:
 
-- ``Annotated``
-- ``AsyncContextManager``
-- ``AsyncGenerator``
-- ``AsyncIterable``
-- ``AsyncIterator``
-- ``Awaitable``
-- ``ChainMap``
-- ``ClassVar``
-- ``Concatenate``
-- ``ContextManager``
-- ``Coroutine``
-- ``Counter``
-- ``DefaultDict``
-- ``Deque``
-- ``final``
-- ``Final``
-- ``Literal``
-- ``NewType``
-- ``NoReturn``
-- ``NotRequired``
-- ``overload``
-- ``OrderedDict``
-- ``ParamSpec``
-- ``ParamSpecArgs``
-- ``ParamSpecKwargs``
-- ``Protocol``
-- ``Required``
-- ``runtime_checkable``
-- ``Self``
-- ``Text``
-- ``Type``
-- ``TypedDict``
-- ``TypeAlias``
-- ``TypeGuard``
-- ``TYPE_CHECKING``
-
-Python 3.7+
------------
-
-- ``get_origin``
-- ``get_args``
-- ``get_type_hints``
-
+- Experimental features
+
+  - ``NotRequired`` (see PEP 655)
+  - ``Required`` (see PEP 655)
+  - ``Self`` (see PEP 673)
+
+- In ``typing`` since Python 3.10
+  
+  - ``Concatenate`` (see PEP 612)
+  - ``ParamSpec`` (see PEP 612)
+  - ``ParamSpecArgs`` (see PEP 612)
+  - ``ParamSpecKwargs`` (see PEP 612)
+  - ``TypeAlias`` (see PEP 610)
+  - ``TypeGuard`` (see PEP 647)
+
+- In ``typing`` since Python 3.9
+
+  - ``Annotated`` (see PEP 593)
+
+- In ``typing`` since Python 3.8
+
+  - ``final`` (see PEP 591)
+  - ``Final`` (see PEP 591)
+  - ``Literal`` (see PEP 586)
+  - ``Protocol`` (see PEP 544)
+  - ``runtime_checkable`` (see PEP 544)
+  - ``TypedDict`` (see PEP 589)
+  - ``get_origin`` (``typing_extensions`` provides this function only in Python 3.7+)
+  - ``get_args`` (``typing_extensions`` provides this function only in Python 3.7+)
+
+- In ``typing`` since Python 3.7
+
+  - ``OrderedDict``
+
+- In ``typing`` since Python 3.5 or 3.6 (see `the typing documentation
```

Comment:
Separating out 3.5 and 3.6 is hard because things were added in patch releases (for example, ChainMap was new in 3.5.4 and 3.6.1). Listing out all the details about that doesn't seem useful.

Reply:
Especially since we don't support 3.5 anymore.